canonical-web-and-design/cn.ubuntu.com
# About

### Done

Completely replaced the /about page with new content
### QA

Check the content against the new copydoc and check styling at all viewports.

https://docs.google.com/document/d/1n9aohemJEeprvoBcd66Hg2fNRV_4f4yiGZWm_zmnP1k/edit#heading=h.8ahslf1aogih
